frontend/src/views/CreditsPage/CreditsPage.jsx

```diff
@@ -0,0 +1,47 @@
+import './CreditsPage.scss';
+
+import { Typography } from '@mui/material';
+import Button from '@mui/material/Button';
+
+import buttonStyles from '../../components/Button/Button.module.scss';
+
+const ButtonsComponent = () => {
+  const developers = [
+    { href: 'https://github.com/Nefariusek', name: 'Szymon' },
+    { href: 'https://github.com/mngweb', name: 'Sylwia' },
+    { href: 'https://github.com/NataliaCichonska', name: 'Natalia' },
+    { href: 'https://github.com/MariaBanaszkiewicz', name: 'Maria' },
+    { href: 'https://github.com/Urszuja', name: 'Ula' },
+    { href: 'https://github.com/GRosza', name: 'Grzegorz' },
+  ];
+  return (
+    <>
+      {developers.map(({ href, name }) => (
+        <Button variant="contained" className={buttonStyles.Button} target="_blank" href={href}>
+          {name}
+        </Button>
+      ))}
+    </>
```

Comment:
to omit this 
`<>` and  `</>`  you can just move element from the CreditsPage component 
```jsx
 <div className="github-buttons">
  
 </div>
 ```